src/kata7.getEmployerRole.js

```diff
@@ -1,3 +1,26 @@
-const getEmployerRole = (employeeName, employees) => {};
+const getEmployerRole = (employeename, employees) => {
+   // const emp=employees.find(person=> person.name===employeename);
+  // return emp.role;
+  const employeeArray= employees.filter(person=> person.name===employeename).map((item)=>{
+   return item.role;
+  })
+   return employeeArray.toString();
+     
+
+}
+
+
+
+
+
+ 
+
+ // const employeeArray= employees.filter(person=> person.name===employeename).map((item)=>{
+   //item.role;
+   //return employeeArray.toString();
+     
+    
+
+
```

Comment:
This is a bit messy, you could rewrite this with better formatting to this:
```
const getEmployerRole = (employeeName, employees) => {
    const employeeArray = employees
        .filter((person) => person.name === employeeName)
        .map((item) => item.role);
    return employeeArray.toString();
};
```
Again, the logic here is identical to what you wrote, I just formatted it for you